### ISSUE TITLE
genvalidity-typed-uuid-0.0.0.2/src/Data/GenValidity/UUID/Typed.hs:10:10: error: Not in scope: type constructor or class ‘GenUnchecked’

### ISSUE THREAD

**Issue by juhp** [NorfairKing/typed-uuid#3]
In Stackage Nightly build:
```
/var/stackage/work/unpack-dir/unpacked/genvalidity-typed-uuid-0.0.0.2-6021d6cc16927f8334e52dabd3a992c746c768f05d924953b2fe7c77dae29888/src/Data/GenValidity/UUID/Typed.hs:10:10: error:  
    Not in scope: type constructor or class ‘GenUnchecked’                                                                                                                               
   |                                                                                                                                                                                     
10 | instance GenUnchecked (UUID a) where                                                                                                                                                
   |          ^^^^^^^^^^^^                                                                                                                                                               
```

**Comment by NorfairKing** [comment]
Aha, I forgot to release a new `genvalidity-typed-uuid` as well. Should be fixed with `genvalidity-typed-uuid-0.1.0.0`.

**Comment by juhp** [comment]
It seems not completely:

```
/var/stackage/work/unpack-dir/unpacked/genvalidity-typed-uuid-0.1.0.0-60338b0f5615da44874a64584b882962877b490ecea2f5dda88c67ba28bd0c4f/bench/Main.hs:18:7: error:
    Variable not in scope: genUncheckedBench :: Benchmark
   |
18 |       genUncheckedBench @(UUID Int),
   |       ^^^^^^^^^^^^^^^^^

/var/stackage/work/unpack-dir/unpacked/genvalidity-typed-uuid-0.1.0.0-60338b0f5615da44874a64584b882962877b490ecea2f5dda88c67ba28bd0c4f/bench/Main.hs:19:48: error:
    Variable not in scope: genUnchecked :: Gen (UUID Int)
   |
19 |       genBench "valid UUID via genUnchecked" ((genUnchecked `suchThat` isValid) :: Gen (UUID Int))
   |                                                ^^^^^^^^^^^^
```

**Comment by NorfairKing** [comment]
Oh bloody hell. Br

**Comment by NorfairKing** [comment]
Should be fixed in `genvalidity-typed-uuid-0.1.0.1`.
